src/bin/nps.js

```diff
@@ -18,6 +24,7 @@ if (argv && psConfig) {
         logLevel: argv.logLevel,
         prefix: argv.prefix,
         scripts: argv.scripts,
+        args: postCommandFlags,
```

Comment:
another alternative to this new flag would be to just concatenate all the args as space separated strings to `argv._[0]`.

It would be the same functionality either way, since we can run a maximum of one script from the cli now.